integration-tests/NAME.IntegrationTests/Constants.cs

```diff
@@ -17,6 +17,11 @@ public static class Constants {
     public static string SpecificRabbitVersion;
     public static string SpecificServiceVersion;
 
+    // Elasticsearch variables
+    public static string LatestElasticHostname;
```

Comment:
Please use the full name of Elasticsearch, should be `LatestElasticsearchHostname`